src/commands/hash.cr

```diff
@@ -35,6 +35,10 @@ module Redis::Commands::Hash
     run command
   end
 
+  def hincrby(key : String, field : String, increment : Int32)
+    run({"hincrby", key, field, increment})
```

Comment:
The spec failure is legit in this case, but the error from the Redis server is pretty opaque, as parser errors often are. It's saying that it's expecting a string (byte marker `$`) but is getting an integer (byte marker `:`). From a higher level, the issue is that the Redis server enforces that all commands from the client be serialized *only* as an array of strings. I've included a suggested diff that should fix the error, and also allows non-32-bit ints.

```suggestion
  def hincrby(key : String, field : String, increment : Int | String)
    run({"hincrby", key, field, increment.to_s})
```

I'll comment more on the other issue because this can be surfaced at compile time.

Reply:
ah, ok. Cool. Thanks!